lib/api/controllers/authController.js

```diff
@@ -144,14 +144,25 @@ class AuthController extends BaseController {
         }
 
         return this.kuzzle.repositories.token.generateToken(
-          response.content, request, options
+          response.content, request.context.connection.id, options
         )
-          .then(token => ({
-            _id: token.userId,
-            jwt: token.jwt,
-            expiresAt: token.expiresAt,
-            ttl: token.ttl
-          }));
+          .then(token => {
+            // @todo unit test
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)